src/scheduler/Scheduler.cpp

```diff
@@ -283,6 +283,11 @@ faabric::util::SchedulingDecision Scheduler::makeSchedulingDecision(
     faabric::Message& firstMsg = req->mutable_messages()->at(0);
     std::string funcStr = faabric::util::funcToString(firstMsg, false);
 
+    // If topology hints are disabled, unset the provided topology hint
+    if (conf.useTopologyHints == "off") {
+        topologyHint = faabric::util::SchedulingTopologyHint::NORMAL;
+    }
```

Comment:
I would change this to check if they're requesting anything other than `NORMAL`, then print a warning if so. We don't want to have this on accidentally.

Reply:
Good point. Now we:
1. Check if env var set
2. If set and hint different than `NORMAL`, print a warning and unset.